sdk/test/trace/simple_processor_test.cc

```diff
@@ -19,7 +19,7 @@ TEST(SimpleProcessor, ToInMemorySpanExporter)
 {
   std::unique_ptr<InMemorySpanExporter> exporter(new InMemorySpanExporter());
   std::shared_ptr<InMemorySpanData> span_data = exporter->GetData();
-  SimpleSpanProcessor processor(std::move(exporter));
+  SimpleSpanProcessor processor(std::unique_ptr<SpanExporter>{exporter.release()});
```

Comment:
How about:

```
InMemorySpanExporter *exporter_ptr = new InMemorySpanExporter();
std::shared_ptr<InMemorySpanData> span_data = exporter_ptr->GetData();
std::unique_ptr<SpanExporter> exporter(exporter_ptr);
```

For exporter, std::unique_ptr<> should always use the abstraction (SpanExporter), not a specific sub class.

This will remove the need to release + create a unique pointer again to go from a sub class to the abstraction.

Reply:
Yes, this also works and will simply the test code, will apply this change, thanks!